### PR TITLE
add messenger bot example with built-in-nlp

### PR DESCRIPTION
**Pull request by kidkcl**

### CODE REVIEW

File: examples/messenger-built-in-nlp/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Messenger hello world
```
Comment (tw0517tw) [inline examples/messenger-built-in-nlp/README.md]: wrong title

File: examples/messenger-built-in-nlp/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Messenger Built-in NLP
+
+## Install and Run
+
+Download this example or clone [bottender](https://github.com/Yoctol/bottender).
+
+```
+curl https://codeload.github.com/Yoctol/bottender/tar.gz/master | tar -xz --strip=2 bottender-master/examples/messenger-hello-world
+cd messenger-hello-world
```
Comment (kpman) [inline examples/messenger-built-in-nlp/README.md]: messenger-built-in-nlp

File: examples/messenger-built-in-nlp/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+const { MessengerBot } = require('bottender');
+const { createServer } = require('bottender/express');
+
+const bot = new MessengerBot({
+  accessToken: '__FILL_YOUR_TOKEN_HERE__',
+  appSecret: '__FILL_YOUR_SECRET_HERE__',
+});
+
+bot.onEvent(async context => {
+  /* eslint-disable no-console */
```
Comment (kpman) [inline examples/messenger-built-in-nlp/index.js]: You can use `// eslint-disable-next-line no-console` for the next line
or `// eslint-disable-line no-console` after console.log.

Do not disable the rule in the whole file.

Comment (tw0517tw) [reply examples/messenger-built-in-nlp/index.js]: That rule should be off from `examples/.eslintrc.js`
https://github.com/Yoctol/bottender/blob/master/examples/.eslintrc.js

Comment (kidkcl) [reply examples/messenger-built-in-nlp/index.js]: Ok~ I removed this line

File: examples/messenger-built-in-nlp/index.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  console.log(context.event.message.nlp.entities);
+  const entities = context.event.message.nlp.entities;
```
Comment (chentsulin) [inline examples/messenger-built-in-nlp/index.js]: With guard statement and destructuring:

```js
const { datetime } = context.event.message.nlp.entities;

if (!datetime) {
  await context.sendText('Not a date or time');
}

const dt = datetime[0];

if (dt.type === 'value') {
  await context.sendText(`Do you mean ${dt.value} ? :)`);
} else if (dt.type === 'interval') {
  const { from, to } = dt;
  await context.sendText(`Do you mean from ${from.value} to ${to.value} ? :)`);
}
```

Comment (chentsulin) [reply examples/messenger-built-in-nlp/index.js]: Do you mean -> Did you mean

**Pull request merged**